website/docs/getting-started/installation.md

```diff
@@ -73,6 +73,16 @@ helm install chaos litmuschaos/litmus --namespace=litmus --set portal.frontend.s
 
 > **Note:** If your Kubernetes cluster isn't local, you may want not to expose Litmus via `NodePort`. If so, remove `--set portal.frontend.service.type=NodePort` option. To connect to Litmus UI from your laptop, you can use `port-forward svc/chaos-litmus-frontend-service 9091:9091`. Then you can use your browser and open `127.0.0.1:9091`.
 
+- Litmus helm chart depends on `bitnami/mongodb`, which uses a mongodb image not supported on ARM. If you want to install Litmus on an ARM-based server, please use a different image and tag as shown below.
```

Comment:
Just a nitpick :

```suggestion
- Litmus helm chart depends on `bitnami/mongodb`, which uses a mongodb image not supported on ARM. If you want to install Litmus on an ARM-based server, please replace the default one with your custom mongodb arm image as shown below.
```

Reply:
Might also be a good idea to leave a hyperlink to the bitnami chart for reference.